sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisProjectsClient.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+[assembly: CodeGenSuppressType("ConversationAnalysisProjectsClientOptions")]
+
+namespace Azure.AI.Language.Conversations
+{
+    [CodeGenClient("ConversationalAnalysisAuthoringClient")]
+    public partial class ConversationAnalysisProjectsClient
+    {
+        /// <summary>
+        /// Initializes a new instance of ConversationAnalysisAuthoringClient.
+        /// </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.api.cognitiveservices.azure.com). </param>
```

Comment:
It does come from the swagger, yes. See:

- https://github.com/Azure/azure-rest-api-specs/blob/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json#L26
- https://github.com/Azure/azure-rest-api-specs/blob/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/common.json#L730

@ChongTang, this is perhaps something we should update in the published swaggers and should be fine since it's only documentation.

@kristapratico it wouldn't surprise me if you have this too then. I could probably use a swagger transform to fix it.

Reply:
Yep you're right. https://github.com/Azure/azure-sdk-for-python/issues/24827 @mshaban-msft FYI

Reply:
@mshaban-msft I used the following transform in my _autorest.md_ with success:

```yaml
# Correct Endpoint parameter description to reference right domain suffix.
- from: swagger-document
  where: $.parameters.Endpoint
  transform: >
    $["description"] = "Supported Cognitive Services endpoint (e.g., https://<resource-name>.cognitiveservices.azure.com)."
```